std/algorithm/searching.d

```diff
@@ -749,36 +749,38 @@ if (isInputRange!R && !isInfinite!R)
     $(D startsWith!pred(haystack, needles)) is $(D true). If
     $(D startsWith!pred(haystack, needles)) is not $(D true) for any element in
     $(D haystack), then $(D -1) is returned.
+    If more than one needle is provided, `countUntil` will wrap the result
+    in a tuple similar to `Tuple!(ptrdiff_t, "steps", ptrdiff_t needle)`
```

Comment:
Did you mean to quote "needle"? Also, why is the tuple "similar to" and not exactly?